profile/README.md

```diff
@@ -1,34 +1,34 @@
-_A modern DSL for API integrations_
+_Let AI connect the APIs for you_
 
 [![npm packages](https://img.shields.io/badge/npm-%40superfaceai-5850ec)](https://www.npmjs.com/org/superfaceai)
 [![Twitter profile](https://img.shields.io/badge/Twitter-%40superfaceai-5850ec)](https://twitter.com/superfaceai)
 [![Discord server](https://img.shields.io/badge/Discord-superface.ai-5850ec)](https://sfc.is/discord)
 
-## Welcome to Superface
+## Superface.ai
 
-API integrations take forever to build. They’re brittle and hard to test. APIs can change on you without notice and break your integrations.
+Integrations, generated code, and API documentation... All suck.
 
-Superface changes this. If your Node.js app depends on multiple API integrations, Superface is the right solution for you!
+If your app depends on multiple API integrations, Superface is the right solution for you.
 
 The motivation behind Superface is described in [this video](https://www.youtube.com/watch?v=BCvq3NXFb94) from APIdays conference.
```

Comment:
I don't think it is adding much value? Maybe a screencast video from @martyndavies would be better?